components/VerifyCredentialResult.yml

```diff
@@ -15,24 +15,116 @@ components:
+              properties:
+                valid:
+                  type: boolean
+                  description: Validation result.
+                  details:
+                    type: string
+                    description: Optional information to provide in the response.
+                input:
+                  type: string
+                  description: The validFrom value.
+            validUntil:
+              type: object
+              description: Results of validating the validUntil property, if present in the VerifiableCredential.
+              properties:
+                valid:
+                  type: boolean
+                  description: Validation result.
+                  details:
+                    type: string
+                    description: Optional information to provide in the response.
+                input:
+                  type: string
+                  description: The validUntil value.
+            credentialSchema:
+              type: array
+              description: Results from validating the credentialSchema objects if any.
+              items:
+                type: object
+                description: Results from validating a credentialSchema object.
+                properties:
+                  valid:
+                    type: boolean
+                    description: Results of validating the credentialSchema object.
+                  details:
+                    type: string
+                    description: Optional information to provide in the response.
+                  input:
+                    type: object
+                    description: The credentialSchema object.
+            credentialStatus:
+              type: array
+              description: Results from validating the credentialStatus objects if any.
+              items:
+                type: object
+                description: Results of validating a credentialStatus object.
+                properties:
+                  value:
+                    type: integer
+                    description: Value of the statusEntry Bit.
+                  valid:
+                    type: boolean
+                    description: Results of validating the credentialStatus object.
+                  details:
+                    type: string
+                    description: Optional information to provide in the response.
+                  input:
+                    type: object
+                    description: The credentialStatus object.
+            proof:
+              type: array
+              description: Results from validating the proof objects if any.
+              items:
+                type: object
+                description: Results from validating a proof object.
+                properties:
+                  valid:
+                    type: boolean
+                    description: Results of validating the proof object.
+                  details:
+                    type: string
```

Comment:
Should this be an object instead everywhere? Is this meant to be a description? If so, it should be "message" probably?